build.gradle

```
@@ -21,7 +21,7 @@ buildscript {
             opensearch_build += "-SNAPSHOT"
         }
 
-        opensearch_java_version = '3.2.0'
+        opensearch_java_version = '3.3.0'
```

This is the Opensearch Java client which is versioned independently of OpenSearch.
```suggestion
        opensearch_java_version = '3.2.0'
```